### PR TITLE
You can now revive dead fishes, bees and mice with a lazarus injector

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
This PR introduces a signal sent to the target when a lazarus injector is being used on them, hence read the title.
EDIT: I've forgotten to add that Tom no longer counts toward the mice population cap now, being unique.

## Why It's Good For The Game
Being items rather than simple/basic mobs, dead mice (without a mind datum), bees and fishes cannot be revived by the lazarus injector. While I agree that spending one for this may be an egregious waste of mining points, it's still fleshing out the feature a bit.

## Changelog

:cl:
add: You can now revive dead fishes, bees and (mindless) mice with a lazarus injector.
balance: Tom, the mouse, no longer counts toward the mice population cap.
/:cl:

### CODE REVIEW

**Comment by Likteer** [comment]
Bees have a similar problem, turning into trash on death.
Both bees and mice also can't be revived through splashing strange reagent on them.

**Comment by Ghommie** [comment]
> Bees have a similar problem, turning into trash on death. Both bees and mice also can't be revived through splashing strange reagent on them.

Noted, I'll take care of that too.

File: code/modules/mob/living/basic/vermin/mouse.dm [hunk 1]
```diff
@@ -250,6 +249,7 @@
 	response_harm_continuous = "splats"
 	response_harm_simple = "splat"
 	gold_core_spawnable = NO_SPAWN
+	contributes_to_ratcap = FALSE
```
[the remark this answers is not in the document]
Comment (Ghommie) [reply code/modules/mob/living/basic/vermin/mouse.dm:252]: This is a change I've briefly mentioned in the "About the Pull request" section and the changelog.

[the remark this answers is not in the document]
Comment (Ghommie) [reply code/modules/mob/living/basic/vermin/mouse.dm:252]: Nah, it's my fault for not having highlighted it enough.

File: code/modules/mining/equipment/lazarus_injector.dm [hunk 2]
```diff
@@ -39,9 +45,12 @@
 		return
 
 	target_animal.lazarus_revive(user, malfunctioning)
+	expende(target_animal, user)
+
+/obj/item/lazarus_injector/proc/expende(atom/revived_target, mob/user)
```
Comment (san7890) [inline code/modules/mining/equipment/lazarus_injector.dm:50]:
```suggestion
/obj/item/lazarus_injector/proc/expend(atom/revived_target, mob/user)
```

you spelled this wrong

File: code/modules/mob/living/basic/vermin/mouse.dm [hunk 3]
```diff
@@ -296,17 +296,38 @@
 	grind_results = list(/datum/reagent/blood = 20, /datum/reagent/consumable/liquidgibs = 5)
 	decomp_req_handle = TRUE
 	ant_attracting = FALSE
+	var/body_color = "gray"
+	var/critter_type = /mob/living/basic/mouse
 	decomp_type = /obj/item/food/deadmouse/moldy
```
Comment (san7890) [inline code/modules/mob/living/basic/vermin/mouse.dm:301]: new bars should go after vars declared on the parent, so both of the new ones you add should be after `decomp_type`. also would be nice if you could add a DMDoc for statistical purposes

**Review by san7890 — CHANGES_REQUESTED** [review]

**Pull request merged**